.travis/script.sh

```diff
@@ -139,7 +139,7 @@ set -x
 
 [ $BUILD_TARGET != posix-32-bit ] || {
     ./bootstrap || die
-    COVERAGE=1 CFLAGS=-m32 CXXFLAGS=-m32 LDFLAGS=-m32 BuildJobs=10 make -f examples/Makefile-posix check || die
+    BuildJobs=99 COVERAGE=1 CFLAGS=-m32 CXXFLAGS=-m32 LDFLAGS=-m32 BuildJobs=10 make -f examples/Makefile-posix check || die
```

Comment:
As previously mentioned, you could just remove `BuildJobs` entirely from this script, now that the default is 10 in this PR.